wgpu-hal/src/lib.rs

```diff
@@ -1365,11 +1434,46 @@ pub trait CommandEncoder: WasmNotSendSync + fmt::Debug {
 
     // compute passes
 
-    // Begins a compute pass, clears all active bindings.
+    /// Begin a new compute pass, clearing all active bindings.
+    ///
+    /// This clears any bindings established by the following calls:
+    ///
+    /// - [`set_bind_group`]
+    /// - [`set_push_constants`]
+    /// - [`begin_query`]
+    /// - [`set_compute_pipeline`]
+    ///
+    /// [`set_bind_group`]: CommandEncoder::set_bind_group
+    /// [`set_push_constants`]: CommandEncoder::set_push_constants
+    /// [`begin_query`]: CommandEncoder::begin_query
+    /// [`set_compute_pipeline`]: CommandEncoder::set_compute_pipeline
```

Comment:
style: Ditto the above feedback for ~~duplicity~~ duplication with `begin_render_pass`.

suggestion:

```suggestion
    /// - [`set_bind_group`](CommandEncoder::set_bind_group)
    /// - [`set_push_constants`](CommandEncoder::set_push_constants)
    /// - [`begin_query`](CommandEncoder::begin_query)
    /// - [`set_compute_pipeline`](CommandEncoder::set_compute_pipeline)
```